src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1296,29 +1293,33 @@ By default, we include the following package managers in each runtime.
 {% /table %}
 
 # Using Appwrite in a function {% #using-appwrite %} 
-Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies. 
-Authenticating with Appwrite is done via an API key or a JWT token. 
-API keys must be generated and exported as an [environment variable](/docs/advanced/self-hosting/environment-variables).
+Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies.
+Integrate Appwrite into your functions using [authentication](/docs/products/auth), which is done via a JWT token or a dynamic API key. 
```

Comment:
> Integrate Appwrite into your functions using [authentication](/docs/products/auth)

This doesn't really make sense because you don't really "integrate using authentication."